samples/subsys/settings/sample.yaml

```diff
@@ -0,0 +1,33 @@
+sample:
+  name: Settings Sample
+
+tests:
+  sample.subsys.settings:
+    tags: settings
+    depends_on: nvs
+    timeout: 10
+    platform_whitelist: qemu_x86
```

Comment:
can we add native_posix and native_posix_64 ?

Reply:
added